pkg/controller/migmigration/task.go

```diff
@@ -574,6 +629,53 @@ func (t *Task) Run() error {
 		if err = t.next(); err != nil {
 			return liberr.Wrap(err)
 		}
+	case CreateDirectVolumeMigration:
+		err := t.createDirectVolumeMigration()
+		if err != nil {
+			return liberr.Wrap(err)
+		}
+		if err = t.next(); err != nil {
+			return liberr.Wrap(err)
+		}
+	case DirectVolumeMigrationStarted:
```

Comment:
I would remove this phase.

Reply:
I think the original purpose here was to provide some sort of feedback to the effect of the dvm process has actually started (vs. the CR only being created), but it's probably unnecessary. The dim process doesn't have an equivalent step. I have one early step where the CR is created, and a later step to wait for completion.

Reply:
Yeah Scott's right here. Intention was to update migmigration with some status reporting once it started prior to moving on but sounds like we may have this resolved with Alay's progress reporting changes anyway.